.changeset/recursive-name-recurses.md

```diff
@@ -0,0 +1,6 @@
+---
+"@postenbring/hedwig-css": patch
+"@postenbring/hedwig-react": patch
```

Comment:
```suggestion
"@postenbring/hedwig-css": minor
"@postenbring/hedwig-react": minor
```

This is adding a new feature, patches for fixing stuff

Reply:
Or ok. I guess we could say that doing the responsive stuff is just fixing 🤷 
Never mind